src/quote/getQuoteAndBuyCoverInputs.ts

```diff
@@ -0,0 +1,161 @@
+import axios, { AxiosError } from 'axios';
+import dotenv from 'dotenv';
+
+import { calculatePremiumWithCommissionAndSlippage } from '../buyCover/calculatePremiumWithCommissionAndSlippage';
+import {
+  CoverAsset,
+  CoverId,
+  DEFAULT_COMMISSION_RATIO,
+  DEFAULT_SLIPPAGE,
+  MINIMUM_COVER_PERIOD,
+  NEXUS_MUTUAL_DAO_TREASURY_ADDRESS,
+  TARGET_PRICE_DENOMINATOR,
+} from '../constants/buyCover';
+import {
+  Address,
+  CoverRouterQuoteResponse,
+  ErrorApiResponse,
+  GetQuoteApiResponse,
+  GetQuoteResponse,
+  IntString,
+  Integer,
+  PoolCapacity,
+} from '../types';
+
+dotenv.config();
+
+type CoverRouterQuoteParams = {
+  productId: Integer;
+  amount: IntString;
+  period: Integer;
+  coverAsset: CoverAsset;
+};
+
+async function getQuoteAndBuyCoverInputs(
+  productId: Integer,
+  coverAmount: IntString,
+  coverPeriod: Integer, // days
+  coverAsset: CoverAsset,
+  coverBuyerAddress: Address,
+  paymentAsset: CoverAsset = coverAsset,
```

Comment:
We don't have support for NXM through the CoverBroker since it adds unnecessary complexity and CoverBroker is for non Nexus users. Not sure if there is point to add it to sdk, because Nexus users can always go to our app and buy cover with less gas